### PR TITLE
Make ConfirmRefillModal use common/Modal. closes #3091

### LINKED ISSUE #3091: [Rx] Refactor ConfirmRefillModal to use same modal pattern as Secure Messaging
See #3088.

~~Then move to `src/js/common`~~ **Update:** Moving is already done. Just need to make `ConfirmRefillModal` use the common component.

### LINKED ISSUE #4349: Change prescriptions modal title color?
In light of department-of-veterans-affairs/messaging-fe#197, should the prescription name change colors as well?

screen shot 2016-12-12 at 4 29 49 pm

If the answer is "no," it's safe to just close this issue.

### PR DESCRIPTION
**Pull request by webinista**
- Added title property to Modal component.
- Changed prescription title color. Closes #4349

### CODE REVIEW

**Comment by U-DON** [comment]
LGTM. Although, it seems that the handling of the visibility in `ConfirmRefillModal` is redundant when returning an empty div, since the Modal component itself takes the visibility prop.

**Comment by webinista** [comment]
It seems redundant, but `prescription` didn't become defined until the user triggers the modal and `this.props.isVisible` is `true`. Rewrote it to check whether `this.props.prescription` is available before using it. Please review again.

File: src/js/rx/components/ConfirmRefillModal.jsx [hunk 1]
```diff
@@ -21,58 +21,54 @@ class ConfirmRefillModal extends React.Component {
...
-          </div>
-        );
-      } else {
+    if (this.props.isLoading) {
+      innerElement = (
+        <LoadingIndicator
+            message="Submitting your refill request..."/>
+      );
+    } else {
+      if (this.props.prescription) {
```
Comment (U-DON) [inline src/js/rx/components/ConfirmRefillModal.jsx]: Can combine this into an `else if`.

File: src/js/rx/components/ConfirmRefillModal.jsx [hunk 2]
```diff
@@ -21,58 +21,54 @@ class ConfirmRefillModal extends React.Component {
...
       }
+    }
 
-      element = (
-        <section className="va-modal rx-modal" id="rx-confirm-refill">
-          <form className="va-modal-inner" onSubmit={this.handleConfirmRefill}>
-            {innerElement}
-          </form>
-        </section>
+    const element = (
```
Comment (U-DON) [inline src/js/rx/components/ConfirmRefillModal.jsx]: Can just return now without assigning first to a variable.

**Review by U-DON — COMMENTED** [review]
LGTM. Suggested some really minor tweaks.

**Pull request merged**